README.md

```diff
@@ -93,6 +77,9 @@ wcb file =
   & Stream.fold Fold.length  -- IO Int
 ```
 
+If you are wondering about the `Internal` module import, see [A Note on
+Internal Modules](#a-not-on-internal-modules) section.
```

Comment:
Broken link. Should be `#a-note-on-internal-modules`.

```suggestion
Please go through [this](#a-note-on-internal-modules) section 
for more details on internal modules.
```

Reply:
fixed already!